content/en/agent/guide/network.md

```diff
@@ -26,6 +26,7 @@ further_reading:
   - [APM][1] data is `trace.agent.`{{< region-param key="dd_site" code="true" >}}
   - [Live Containers][2] data is `process.`{{< region-param key="dd_site" code="true" >}}
   - [Logs][3] data is `agent-intake.logs`{{< region-param key="dd_site" code="true" >}} for TCP traffic, `agent-http-intake.logs.`{{< region-param key="dd_site" code="true" >}} in HTTP. Review the list of [logs endpoints][4] for more information.
+  - [Orchestrator Resources][5] data is `orchestrator.datadoghq.com`
```

Comment:
Just need confirmation if this is `.com` for both US & EU. If it changes, I can add the partial which will change this to `.eu` whenever someone uses the EU region selector.

Reply:
good catch, it's .com for US et .eu for UE.